Tasks/PowerShellOnTargetMachines2/RunPowerShellScriptJob.ps1

```diff
@@ -19,7 +19,9 @@ $ExecutePsScript = {
             [string] $toolPath,
             [string] $toolArgs
         )
-        Invoke-Expression "& '$($toolPath.Replace('"', '').Replace("'", "''"))' $($toolArgs.Trim())"
+        $command = "& '$($toolPath.Replace('"', '').Replace("'", "''"))' $($toolArgs.Trim())"
+        Write-Host "##[command]$command"
+        Invoke-Expression $command
```

Comment:
If system.debug is set to true, we will have to set the verbosity in remote execution so that all write-verbose during execution are displayed.

Reply:
i am setting the verbose preference to continue explicitly so that verbose data is logged every time. I think we should output it regardless of what the value of system.debug is. @RoopeshNair to take a call on this.